vault/__init__.py

```diff
@@ -20,20 +21,36 @@ def index():
 
     @app.route("/unlock", methods=["POST"])
     def unlock():
-
         if request.method == "POST":
             domain = validate(request.form["websiteURL"]).geturl()
+            dictionary = json.loads(app.config["ALLOWED_SITES"])
+            wp_path = dictionary[domain]
+            if domain not in dictionary:
+                return redirect("https://subscriptionwebsitebuilder.co.uk", code=301)
             if wordpress_login_success(domain) is False:
                 return "Wrong Password or Username"
-            subprocess.run(app.config["PATH_TO_UNLOCK_SCRIPT"], shell=True)
+            print(wp_path)
+            subprocess.run(
+                app.config["PATH_TO_UNLOCK_SCRIPT"] + " " + wp_path, shell=True
+            )
             return render_template("unlock.html")
         return render_template("index.html")
 
     @app.route("/lock", methods=["POST"])
     def lock():
-        print("your site is locked")
-        subprocess.run(app.config["PATH_TO_LOCK_SCRIPT"], shell=True)
-        return render_template("lock.html")
+        if request.method == "POST":
+            domain = validate(request.form["websiteURL"]).geturl()
+            dictionary = json.loads(app.config["ALLOWED_SITES"])
+            wp_path = dictionary[domain]
+            if domain not in dictionary:
```

Comment:
Nice, we can use this to redirect to account creation when webaddress does not exist 

Please may you make it redirect to a new route, then, redirect to e.g. /sign-up instead of subscriptionwebsitrbuilder?